src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Figure[] figures = new Figure[]{

```

Comment:
```suggestion
        FigureSupplier supplier = new FigureSupplier();
        Figure[] figures = new Figure[6];
```

create constant for 6